allennlp/service/servable/models/bidaf.py

```diff
@@ -33,24 +34,28 @@ def predict_json(self, inputs: JSONDict) -> JSONDict:
         return output_dict
 
     @classmethod
-    def from_params(cls, params: Params) -> 'BidafServable':
-        glove_path = params.pop("glove_path")
-        constants.GLOVE_PATH = glove_path
-
-        tokenizer = Tokenizer.from_params(params.pop("tokenizer"))
+    def from_config(cls, config: Params) -> 'BidafServable':
+        tokenizer_params = config.pop("tokenizer", None)
+        if tokenizer_params is None:
+            tokenizer = WordTokenizer()
+        else:
+            tokenizer = Tokenizer.from_params(tokenizer_params)
 
         token_indexers = {}
-        token_indexer_params = params.pop('token_indexers')
+        token_indexer_params = config["dataset_reader"].pop('token_indexers')
```

Comment:
I _think_ it's safe and fast to just instantiate the `DatasetReader` itself here.  Then you can pull out the `_tokenizer` and `_token_indexer` fields directly from the reader, and have 3-4 lines of code, instead of 10, for this.  It'd be reading the internal fields of the `DatasetReader`, but I don't have a huge problem with that in this case.

If you _do_ really want to keep it this way, the tokenizer params are inside "dataset_reader", not at the top level - we just use the default in BiDAF, so it's not obvious where they would have shown up.